examples/schema-arrays/expected.txt

```diff
@@ -0,0 +1,18 @@
+jobs: []
+***
+jobs:
+- name: capi-k8s-release
+  github_release: false
+  vendir_github_release: false
+  github_org: cloudfoundry
+  build_dir: ""
+- name: eirini-release
+  github_org: cloudfoundry-incubator
+  build_dir: build/eirini
+  github_release: true
+  vendir_github_release: true
+- name: metric-proxy
+  github_release: true
+  github_org: cloudfoundry
+  vendir_github_release: true
+  build_dir: ""
```

Comment:
Is this file being used somewhere? I can't seem to find that usage.

Reply:
The `expected.txt` files are now being used [here](https://github.com/vmware-tanzu/carvel-ytt/pull/414/files#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99R146-R161)